CHANGELOG.md

```diff
@@ -3,9 +3,11 @@
 This changelog keeps track of all changes to the Packs SDK. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
 ### Changed
 
 - **Future Breaking Change** Test framework will no longer normalize object output using the schema. (This will continue to work when executing the pack within Coda)
+- Added support for making `HEAD` HTTP requests using the fetcher.
```

Comment:
Nitpick: Should this be under an "Added" heading instead, since we're adding a new feature?

Reply:
oops good catch